service/service.go

```diff
@@ -18,13 +17,14 @@ import (
 )
 
 type Service interface {
-	GRPCClient() *panacea.GRPCClient
-	EnclaveInfo() *sgx.EnclaveInfo
-	OracleAcc() *panacea.OracleAccount
-	OraclePrivKey() *btcec.PrivateKey
-	Config() *config.Config
-	QueryClient() panacea.QueryClient
-	IPFS() *ipfs.IPFS
+	GetGRPCClient() panacea.GRPCClient
+	GetEnclaveInfo() *sgx.EnclaveInfo
+	GetSgx() sgx.Sgx
+	GetOracleAcc() *panacea.OracleAccount
+	GetOraclePrivKey() *btcec.PrivateKey
+	GetConfig() *config.Config
+	GetQueryClient() panacea.QueryClient
+	GetIPFS() ipfs.IPFS
```

Comment:
functions typically use `Get` to get a value of some kind, so I'll rename these functions.